### PR TITLE
[PR2] DEV-88 Charts Sidebar and ChartMenu

### PR DESCRIPTION
**Pull request by smorsua**
This PR adds the sidebar that is used to drag and drop measurements into the chart space. It also adds the component that wrapps the Chart and the Sidebar. The drag and drop is done using the `DataTransfer` API

[screen-capture (2).webm](https://user-images.githubusercontent.com/114338434/207666662-c115eb1d-e78f-492c-bc71-b50035e0477b.webm)

### CODE REVIEW

**Review by jmaralo — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -30,16 +30,16 @@ function App() {
     let messageSocket = messageService.createMessageWebSocket();
     let connectionSocket = connectionService.createConnectionsSocket();
 
-    const intervalId = setInterval(() => {
-      dispatch(updateMessages(getRandomMessage(warningMessages)));
-      dispatch(updateMessages(getRandomMessage(faultMessages)));
-    }, 1000);
+    // const intervalId = setInterval(() => {
```
Comment (Losina24) [inline src/App.tsx:33]: if u r not using it, delete it. You can rollback using git at any moment

File: src/components/ChartMenu/ChartMenu.tsx [hunk 2]
```diff
@@ -115,6 +67,12 @@ export const ChartMenu = () => {
               key={chartElement.id}
               chartElement={chartElement}
               handleDropOnChart={handleDropOnChart}
+              removeElement={() => {
+                removeElement(chartElement.id);
```
Comment (Losina24) [inline src/components/ChartMenu/ChartMenu.tsx:71]: I think u dont need to put the curly braces: removeElement={ () => removeElement(chartElement.id) }

File: src/components/ChartMenu/Sidebar/BoardItem/BoardItem.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  name: string;
+  packetNodes: TreeNode;
+};
+
+export const BoardItem = ({ name, packetNodes }: Props) => {
+  return (
+    <div id={styles.wrapper}>
+      <div id={styles.header}>{name}</div>
+      <div id={`${styles.packetList} ${styles.treeNode}`}>
+        {Object.entries(packetNodes).map(([name, measurementNodes]) => {
```
Comment (Losina24) [inline src/components/ChartMenu/Sidebar/BoardItem/BoardItem.tsx:15]: move this to a new method, u should not mix logic and view:

```
const generatePacketItems = (packetNodes): JSX.Element => (
    Object.entries(packetNodes).map(([name, measurementNodes]) => (
            <PacketItem
              key={name}
              name={name}
              measurementNodes={measurementNodes!} />
    )
)

export const BoardItem = ({ name, packetNodes }: Props): JSX.Element => (
    <div id={styles.wrapper}>
      <div id={styles.header}>{name}</div>
      <div id={`${styles.packetList} ${styles.treeNode}`}>
       { generatePacketItems(packetNodes) }
       </div>
    </div>
)
```

It is much more readable

**Review by Losina24 — COMMENTED** [review]
Can improve some things

**Review by lipezaballa — APPROVED** [review]

**Pull request merged**